api/validate/validate.py

```diff
@@ -0,0 +1,7 @@
+class Validate:
```

Comment:
Not sure if bringing this into a separate class+method this is useful. 
Typically all the code is specified inside the view (or more basically in a reusable method of view set)